src/AppInstallerCLICore/ExecutionContextData.h

```diff
@@ -51,10 +51,30 @@ namespace AppInstaller::CLI::Execution
         Max
     };
 
+    // Contains all the information needed to install a package.
+    // This is used when installing multiple packages to pass all the
+    // data to a sub-context.
     struct PackageToInstall
     {
+        PackageToInstall(
+            std::shared_ptr<Repository::IPackageVersion>&& packageVersion,
+            std::shared_ptr<Repository::IPackageVersion>&& installedPackageVersion,
+            Manifest::Manifest&& manifest,
+            Manifest::ManifestInstaller&& installer,
+            Manifest::ScopeEnum scope = Manifest::ScopeEnum::Unknown)
+            : PackageVersion(packageVersion), InstalledPackageVersion(installedPackageVersion), Manifest(std::move(manifest)), Installer(std::move(installer)), Scope(scope) { }

```

Comment:
>packageVersion [](http://example.com/codeflow?start=29&length=14)

sorry, just noticed the 2 shared_ptr are rvalue too, so std::move on these 2 as well